libnavigation-core/src/test/java/com/mapbox/navigation/core/lifecycle/CarAppLifecycleOwnerTest.kt

```diff
@@ -371,7 +369,6 @@ class CarAppLifecycleOwnerTest {
             onResume(carAppLifecycleOwner)
         }
         carAppLifecycleOwner.activityLifecycleCallbacks.apply {
-            val activity: Activity = mockActivity()
```

Comment:
wait, does this break something? it needs to be a separate instance for the test right?

Reply:
never mind

Reply:
this activity should not be created before paused/stopped/destroyed

so it was an issue in the test exposed by the new fix